common/utils.py

```diff
@@ -42,3 +43,14 @@ def render_to_template(template_name, context=None):
         context = dict()
     template = get_template(template_name)
     return template.render(context)
+
+
+def get_system_user():
+    """
+    Return the user object used by the system
+
+    :return: the User object associated with the system
+    :rtype: User
+    """
+    user, _ = get_user_model().objects.get_or_create(username='chef_robot')
+    return user
```

Comment:
how does user, _ work?

Reply:
the return of `get_or_create` is a tuple containing the object, and whether or not it was created, like this:

`(User, True)`.

If the user already existed, the above will be `False`, if it was created, it'll be `True`. That's so you can do _other stuff_ based on if something was created. For us, we don't care, we just want it to exist. So, we're assigning that Boolean value to `_` which is a python convention for a variable that contains garbage and isn't used.

Pycharm knows to ignore `_` and `__` variables when linting in the IDE, so it won't flag that as "This variable is not used" or anything like that.